Models/Products.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace BangazonAPI.Models
+{
+    public class Products
+    {
+        [Key]
+        public int Id { get; set; }
+
+        
+
+        [Required]
+        public int TypeId { get; set; }
+        public ProductTypes ProductId { get; set; }
+        public int CustomerID { get; set; }
+        public Customers Customer { get; set; }
+        public string FirstName { get; set; }
+        public int? Price { get; set; }
+        public string Title { get; set; } // ? means that the variable can be null
+        public string Description { get; set; }
+        public int? Quantity { get; set; }
```

Comment:
this doesn't need to be null correct? They need to have at least 1. Could be wrong on this so I will make this a question